builder/kubernetes/action.go

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package builder
```

Comment:
all the go files under kubernetes are strictly used only by kubernetes builder?

Reply:
yes